sources/packages/web/src/services/http/dto/Common.dto.ts

```diff
@@ -16,16 +18,26 @@ export interface InstitutionPrimaryContactAPIOutDTO {
  * a generic address interface that is used extend or used as nested type
  * for both IN and OUT DTOs
  */
-export interface AddressDetailsFormAPIDTO {
+export class AddressDetailsFormAPIDTO {
```

Comment:
APIInDTO?

Reply:
Please see the comments, they are from 14 months ago. I will not be changing it.
![image](https://github.com/bcgov/SIMS/assets/61259237/8accea02-1c0f-492c-9877-1859724ed881)

Reply:
I didn't notice that the name just came from an existing interface and I'm ok with the name being as it is.